CanalTP/Chaos
# Bugfix impact search sortby

# Description

This PR make impact search sortby impactApplicationEndDate impactUpdatedAt and severityPriority works

## Pull Request type (optional)

This is a bug fix

## How to test

Here are the following steps to test this pull request:

- POST http://127.0.0.1:5000/impacts/_search with `"by":"impactApplicationEndDate"`, `"by":"impactUpdatedAt"`, `"by":"severityPriority"`
Example: ```{"items_per_page":3,"application_status":["past"],"sort":{"by":"impactApplicationEndDate","order":"desc"},"with_disruption_types":["unexpected"],"depth":2,"start_page":1,"current_time":"2021-09-08T12:37:44Z"}```
- The results will be well ordered
- If your impacts are `unexpected`, see your TRV2 IHM, the `ongoing` part and the `closed` part will be well ordered
